docs/basic-serialization.md

```diff
@@ -437,10 +437,10 @@ fun main() {
 
 > You can get the full code [here](../guide/example/example-classes-09.kt).
 
-It produces the following output, which has `language` property, even though its value is equal to the default one.
+It produces the following output, which does not have `language` property, because its value is equal to the default one.
```

Comment:
Please add a short sentence explaining the rationale of this behaviour

Reply:
Friendly reminder

Reply:
Yeah thanks